### DISCUSSION TITLE
Discussion: Mar 25, 2023 DataFusion meetup at the DataCouncil March 2024 in Austin TX

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#8152] [Ideas]
I would like to offer to organize a face to face meetup for the DataFusion community. 

## Proposed dates:

* **Date**: Monday March 25, 2024 or Thur March 28 2024
* **Venue**: Somewhere near the  [DataCouncil '24](https://www.datacouncil.ai/) conference venue, [AT&T Hotel & Conference Center in Austin TX, USA](https://www.google.com/maps/place/AT%26T+Hotel+and+Conference+Center/@30.2816295,-97.7404408,15z/data=!4m2!3m1!1s0x0:0x7ef52b1ad3321879?sa=X&ved=2ahUKEwin17CNyK78AhWjPH0KHVC3Cq4Q_BJ6BAh0EAc)

I am contemplating attending (and have submitted a proposal to present) at the [DataCouncil '24](https://www.datacouncil.ai/) conference March 26th - 28th, 2024. Given I think several other contributors to DataFusion may also be attending, I think this would be a good time to organize a meetup for anyone who may be interested.

It is important to note that we will not "decide" anything at the meeting, and will of course follow the same process as we do for other consensus lead decisions, including publicly posting all materials, and providing a video link/recordings for anyone who would like to attend but can not travel.

## Goals:

* Build community, discuss common goals and challenges
* Brainstorm ways to grow and nurture the community (especially the non-feature focused aspects)

## Potential agenda items

* What are people using DataFuson for ? Where would they like to see it go?  (e.g. https://github.com/apache/arrow-datafusion/discussions/6441)
* Discussion: Barriers to adoption and brainstorm how to remove them (and to get more users)
* Discussion: Barriers to contribution and brainstorm how to get more people to contribute (both features but also non features like code review, documentation, and blog posts)
* Discussion "Non technical roadmap" like better test coverage, documentation, easier to use APIs, etc.
* Discussion: How shall we evangelize the project more?
* Feature roadmap: Use the event as a catalyst to write down a community feature roadmap

**Comment by andygrove** [comment]
This sounds great. I plan on attending. Thanks for organizing this @alamb.

**Comment by thinkharderdev** [comment]
Nice! I'm hoping to be there as well.

**Comment by stormasm** [comment]
I will not be able to attend but I think this is a great idea !  Spending time with people in real life is very important 😄 

And if an opportunity like this presents itself then getting together is awesome.

Sorry I will miss the gathering...

**Comment by comphead** [comment]
Sounds really great, are you planning to have a video recording and photos?

**Reply by alamb** [reply]
>  providing a video link/recordings for anyone who would like to attend but can not travel.

Yes, I will do so to the best of my ability

**Comment by avantgardnerio** [comment]
I'm pretty sure I will attend, if not the conference than at least this meetup and any associated happy hour type fun.

**Comment by yukkit** [comment]
This sounds really great, and I'd love to participate, but I'm in a different country, so unfortunately, I won't be able to join in person.

**Comment by Dandandan** [comment]
Sounds like a  fantastic plan! I expect a child around yhe end of January so March is a bit too early for me.

**Comment by ozankabak** [comment]
I am in! I will be attending Data Council and would love to see this happen.

**Comment by wesm** [comment]
I am submitting a talk to the conference and plan to be there whether it gets accepted or not — look forward to meeting/seeing many of you there.

**Comment by alamb** [comment]
Save the date for  Monday March 25, 2024  -- we'll try and meet prior to the main conference so as not to be totally exhausted.

**Comment by txbm** [comment]
I live in Austin and am prototyping with DataFusion. Should be able to make it!

**Comment by alamb** [comment]
Update here is I hope to have a more final confirmation / invitation mid-December (I am working to finalize the venue and budget, etc)

**Comment by alamb** [comment]
Official inviation is here: https://github.com/apache/arrow-datafusion/discussions/8522 please RSVP as described there. Let's call this discussion closed!
